core/templates/pages/release-coordinator-page/navbar/release-coordinator-navbar.component.ts

```diff
@@ -86,11 +82,22 @@ export class ReleaseCoordinatorNavbarComponent implements OnInit {
           '/profile/<username>', {
             username: this.username
           }));
+      let profileImageUrls = this.userService.getProfileImageDataUrlAsync(
+        this.username);
+      this.profilePicturePngDataUrl = profileImageUrls[0];
+      this.profilePictureWebpDataUrl = profileImageUrls[1];
```

Comment:
Can we simplify this a bit by destructuring https://www.geeksforgeeks.org/how-tuple-destructuring-works-in-typescript/?

Reply:
ditto elsewhere

Reply:
Done